.github/workflows/docker-openmetadata-db.yml

```diff
@@ -40,5 +44,5 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name == 'release' }}
           # Update tags before every release
-          tags: 'openmetadata/db:0.10.0,openmetadata/db:latest'
-          file: ./docker/local-metadata/Dockerfile_mysql
\ No newline at end of file
+          tags: 'openmetadata/db:${{ github.event.inputs.tag }},openmetadata/db:latest'
```

Comment:
`github.event.inputs.tag` will be null in case of Github CI triggered by Release published event!